frontend/public/components/graphs/stack.tsx

```diff
@@ -110,8 +111,9 @@ export const StackChart: React.FC<AreaChartProps> = ({
   return (
     <PrometheusGraph className={className} ref={containerRef} title={title}>
       {data?.[0]?.length ? (
-        <PrometheusGraphLink query={query}>
+        <PrometheusGraphLink query={query} ariaChartLabel={ariaChartLabel}>
           <Chart
+            ariaTitle={ariaChartLabel}
```

Comment:
Shouldn't the link and chart have different aria labels?

cc @jessiehuff

Reply:
The link will get an `aria-label`, the chart get's a title element (it's the way victory charts work):
![image](https://user-images.githubusercontent.com/12733153/104652680-7c3edb80-5687-11eb-8b98-efb4cbe50cc3.png)

Reply:
But I'd expect the aria-label of the link not to describe the chart. It should describe what the link does like "View detailed metrics" or similar.

Reply:
`ariaChartLabel` and `title` were being used interchangeably.  Renamed `ariaChartLabel` to `ariaChartLinkLabel` and used it exclusively in `<PrometheusGraphLink`.  Updated `<Chart ariaTitle=...` to always use `title` and not `ariaChart[Link]Label`.  

Standardized `ariaChartLinkLabel`'s to include a verb and more description, such as 'View Memory usage chart details', whereas `title` remains a standard title, ex: `Memory usage`.

The updated usage of these props:
```
<PrometheusGraphLink query={query} ariaChartLabel={ariaChartLinkLabel}>
          <Chart
            ariaTitle={title}
```

Also had to introduce a `showTitle` prop. for the Area charts shown on Cluster Dashboard where we don't want to display a title with each chart as that would show double titles as the chart title is displayed on the left in the dashboard Utilization Card; but we do want to set/pass `title` so it will be used as a `<title ...` element in the `<svg ..` chart for a11y purposes.